api/config/config_local.json

```diff
@@ -143,6 +143,7 @@
     "usersPerSynchronizeAccessTask": 50
   },
   "egressAlertRemediationPolicy": {
+    "notifyFromEmail": "workbench-notificatio-aaaaekogch7vg3c6hi4sgyx4le@pmi-engteam.slack.com",
```

Comment:
I think we need to notify security teams.
So We may need a mailing list for those people, would there be a cc config or use the notifyFromEmail.
I am not sure if we can make the email send from a mailing list, so it may need to be in cc field